openmdao/solvers/test/test_newton.py

```diff
@@ -65,6 +65,7 @@ def test_sellar_derivs_with_Lin_GS(self):
         prob.root = SellarDerivatives()
         prob.root.nl_solver = Newton()
         prob.root.ln_solver = LinearGaussSeidel()
+        prob.root.ln_solver.options['maxiter'] = 15
```

Comment:
Good question.  I lowered maxiter to 2 to avoid the new exception and it passes.  I also went back and commented out the new exception and put the maxiter back to 1 and it does pass.  I have no idea why though.  Did you write this test or was it justin?